### PR TITLE
rust: fix lint warnings about is same then else

### PR DESCRIPTION
**Pull request by Rai0Catodic0**
Ticket: [4609](https://redmine.openinfosecfoundation.org/issues/4609)

Make sure these boxes are signed before submitting your Pull Request -- thank you.

- [ :white_check_mark: ] I have read the contributing guide lines at https://redmine.openinfosecfoundation.org/projects/suricata/wiki/Contributing
- [ :white_check_mark: ] I have signed the Open Information Security Foundation contribution agreement at https://suricata.io/about/contribution-agreement/
- [ ] I have updated the user guide (in doc/userguide/) to reflect the changes made (if applicable)

Previous PR: #8008 

Describe changes:
- Fix commit message

### CODE REVIEW

File: rust/src/nfs/nfs.rs [hunk 1]
```diff
@@ -1668,11 +1668,8 @@ pub unsafe extern "C" fn rs_nfs_tx_get_alstate_progress(tx: *mut std::os::raw::c
                                                   -> std::os::raw::c_int
 {
     let tx = cast_pointer!(tx, NFSTransaction);
-    if direction == Direction::ToServer.into() && tx.request_done {
-        //SCLogNotice!("TOSERVER progress 1");
-        return 1;
-    } else if direction == Direction::ToClient.into() && tx.response_done {
-        //SCLogNotice!("TOCLIENT progress 1");
+    if (direction == Direction::ToServer.into() && tx.request_done) || (direction == Direction::ToClient.into() && tx.response_done) {
+        //SCLogNotice!("TOSERVER progress 1"); or SCLogNotice!("TOCLIENT progress 1");
```
Comment (jasonish) [inline rust/src/nfs/nfs.rs:1672]: On second look, I think these should be kept separate.  If you revert this change, uncomment `SCLogNotice`, then convert to `SCLogDebug`, then this lint should be cleared up.  @victorjulien Any reason to keep this debug code around?

**Comment by jasonish** [comment]
Also, the commit message still doesn't appear to be fixed.

**Comment by inashivb** [comment]
Closing in favor of newer revs.
